www/src/styles/global.css

```diff
@@ -155,7 +155,7 @@
   }
 
   .markdown code {
-    @apply break-words lg:break-normal;
+    @apply break-all lg:break-normal;
```

Comment:
As per https://github.com/t3-oss/create-t3-app/issues/744#issuecomment-1316444285, this is not the problem for the layout shift, it's the iframes that are unresponsive